src/Shared/PackageManagers/NuGetProjectManager.cs

```diff
@@ -24,8 +24,7 @@ public class NuGetProjectManager : BaseProjectManager
         [System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0044:Add readonly modifier", Justification = "Modified through reflection.")]
         public static string ENV_NUGET_ENDPOINT_API = "https://api.nuget.org";
         public static string ENV_NUGET_ENDPOINT = "https://www.nuget.org";
-
-        private readonly string NUGET_DEFAULT_REGISTRATION_ENDPOINT = "https://api.nuget.org/v3/registration5-gz-semver2/";
+        public static string NUGET_DEFAULT_REGISTRATION_ENDPOINT = "https://api.nuget.org/v3/registration5-gz-semver2/";
```

Comment:
This seems like it should remain readonly.  Alternatively, you might be able to make it const.